code/modules/mob/living/carbon/xenomorph/abilities/lurker/lurker_powers.dm

```diff
@@ -278,6 +278,9 @@
 		to_chat(xeno, SPAN_XENOHIGHDANGER("You can only headbite an unconscious, adjacent target!"))
 		return
 
+	if(xeno.stat == DEAD)
```

Comment:
should they be able to do this while unconscious, either?

Reply:
Added a check for that just now.